<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item rowspan="2" row="0" column="1" >
     <widget class="QTextBrowser" name="textBrowser" />
    </item>
    <item row="0" column="0" >
     <widget class="QToolBox" name="toolBox" >
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_1" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>345</width>
         <height>176</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Example 1</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QGroupBox" name="groupBox_2" >
          <property name="title" >
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" colspan="2" >
            <widget class="QScrollBar" name="horizontalScrollBar" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>171</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="1" >
            <widget class="QDial" name="dial" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>345</width>
         <height>160</height>
        </rect>
       </property>
       <attribute name="label" >
        <string>Example 2</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QGroupBox" name="groupBox" >
          <property name="title" >
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="1" >
            <spacer>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" >
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0" >
            <layout class="QGridLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item row="2" column="1" >
              <widget class="QRadioButton" name="radioButton_3" >
               <property name="text" >
                <string>RadioButton</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" >
              <widget class="QPushButton" name="pushButton_3" >
               <property name="text" >
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" >
              <widget class="QRadioButton" name="radioButton_2" >
               <property name="text" >
                <string>RadioButton</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" >
              <widget class="QPushButton" name="pushButton_2" >
               <property name="text" >
                <string>PushButton</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" >
              <widget class="QRadioButton" name="radioButton_1" >
               <property name="text" >
                <string>RadioButton</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0" >
              <widget class="QPushButton" name="pushButton_1" >
               <property name="text" >
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3" >
       <attribute name="label" >
        <string>Example 3</string>
       </attribute>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QGroupBox" name="groupBox_3" >
          <property name="title" >
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" >
           <property name="margin" >
            <number>9</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item row="0" column="0" >
            <widget class="QLineEdit" name="lineEdit" >
             <property name="text" >
              <string>Try to mark some of this text, or change it if you like.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label" >
             <property name="text" >
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>20</width>
        <height>171</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExamples" >
    <property name="title" >
     <string>Examples</string>
    </property>
    <addaction name="actionShow_Example_1" />
    <addaction name="actionShow_Example_2" />
    <addaction name="actionShow_Example_3" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionExit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuExamples" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionShow_Example_1" >
   <property name="text" >
    <string>Show example 1</string>
   </property>
  </action>
  <action name="actionShow_Example_2" >
   <property name="text" >
    <string>Show Example 2</string>
   </property>
  </action>
  <action name="actionShow_Example_3" >
   <property name="text" >
    <string>Show Example 3</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_1</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>59</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel" >
     <x>144</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_2</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>57</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel" >
     <x>123</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_3</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>53</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel" >
     <x>119</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>dial</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>60</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel" >
     <x>287</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>356</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
